imap/ctl_cyrusdb.c

```diff
@@ -306,9 +306,21 @@ int main(int argc, char *argv[])
     char *msg = "";
     int i, rotated = 0;
 
-    r = r2 = 0;
+    /* keep this in alphabetical order */
+    static const char *const short_options = "C:crx";
```

Comment:
With `static const char short_options[]` the variable will be put in a read-only segment in the final executable.

Reply:
Is that a problem?

Reply:
Oh, I misread: you were suggesting that I should change it to that.  That's a good suggestion, thanks.